src/main/kotlin/net/atos/zac/zoeken/IndexingService.kt

```diff
@@ -327,9 +327,12 @@ class IndexingService @Inject constructor(
     private fun logTypeMessage(objectType: ZoekObjectType, message: String) =
         LOG.info("[$objectType] $message")
 
-    private fun logTypeError(objectType: ZoekObjectType, error: Throwable) =
+    private fun logTypeWarning(objectType: ZoekObjectType, error: Throwable) =
```

Comment:
Which `type` are you referring to? I don't understand the word `type` here. Why not just `logIndexingError` ? The caller of this function is not interested whether it is logged as at the 'WARNING' log level. The function is about logging an error during indexing right?

Reply:
Not sure we even need this function at all to be honest. It only confuses me. I would just log right there where the issue was.

Reply:
Removed log functions